omod/src/main/resources/config.xml

```diff
@@ -80,6 +80,12 @@
 		<description>Specifies role for add provider to XDS documents</description>
 	</globalProperty>
 
+	<globalProperty>
+		<property>xdssender.openmrs.moduleUsedToDetermineSoftwareVersion</property>
+		<defaultValue>isanteplusreports</defaultValue>
```

Comment:
shouldn't this just be isanteplus itself by default?

Reply:
This module name was given by Nathaelf but I can change this to isanteplus.